### DISCUSSION TITLE
Is there any pattern for being able to use mutations and not serializing typename__ for deeply nested classes?

### DISCUSSION THREAD

**Discussion by jzeimen** [koxudaxi/datamodel-code-generator#1971] [Q&A]
Like the question says. I am constructing a deeply nested object using the types generated by the datamodel-code-generator. This will go into a mutation, however it generated the `typename__` fields. Graphene is telling me my type doesn't have the "__typename" field. Just seems like there should be an easy default way to be able to use these generated classes for mutations too but maybe I'm not seeing it?

**Comment by koxudaxi** [comment]
You can exclude `__typename` from serialization using `model.model_dump(exclude={"typename__"})` or `model.model_dump(exclude_defaults=True)`.
